tests/system/action/motion/test_reset_state.py

```diff
@@ -124,7 +126,7 @@ def test_reset_state_correct_number_value(self) -> None:
         model = self.get_model("motion/22")
         assert model.get("state_id") == 76
         assert model.get("number_value") == 23
-        assert model.get("created") == 1111111
+        assert model.get("workflow_timestamp") == 1111111
```

Comment:
must be None, see changed action description of motion.reset_state

Reply:
Done. Set to `is None`.